.circleci/config.yml

```diff
@@ -256,11 +256,11 @@ jobs:
           at:  /go
       - run: make sync
       - run: bin/testEnvRootMinikube.sh start
-      - run:
-          no_output_timeout: 20m
-          when: always
-          command: |
-                make test/local/cloudfoundry/e2e_pilotv2
+      # - run:
+      #     no_output_timeout: 20m
+      #     when: always
+      #     command: |
+      #           make test/local/cloudfoundry/e2e_pilotv2
```

Comment:
why is this commented out?

Reply:
Because this was added in Error.. Costin's lgtm merged my PR before I could remove it. And this step always fails, causing other tests to fail